cloudflare_workers/index.ts

```diff
@@ -1,4 +1,5 @@
-import { Hono } from 'hono/tiny'
```

Comment:
keep tiny here, OpenAPIHono is not required here

Reply:
Yes it IS required. 

I call the following
- `app.openAPIRegistry`
- `app.doc`